src/components/SelectFromQuery/SelectFromQuery.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+export function SelectFromQuery({
```

Comment:
```suggestion
export function SelectDepartmentFromQuery({
```
?

Reply:
This component is used to changing department of user.  I think more sensible name can be  `changeDepartmentComponent`

Reply:
Okay, let it be so